github/misc.go

```diff
@@ -83,22 +83,22 @@ func (c *Client) ListEmojis(ctx context.Context) (map[string]string, *Response,
 	return emoji, resp, nil
 }
 
-// Conduct represents metric that contains name, key, url, body.
-type Conduct struct {
+// CodeOfConduct represents a code of conduct
+type CodeOfConduct struct {
 	Name *string `json:"name,omitempty"`
 	Key  *string `json:"key,omitempty"`
 	URL  *string `json:"url,omitempty"`
-	BODY *string `json:"body,omitempty"`
+	Body *string `json:"body,omitempty"`
 }
 
-func (c *Conduct) String() string {
+func (c *CodeOfConduct) String() string {
 	return Stringify(c)
 }
 
 // ListCodesOfConduct returns all codes of conduct.
 //
-// https://developer.github.com/v3/codes_of_conduct/
-func (c *Client) ListCodesOfConduct(ctx context.Context) ([]*Conduct, *Response, error) {
+// GitHub API docs: https://developer.github.com/v3/codes_of_conduct/
```

Comment:
You can update the URL here to be more specific:

https://developer.github.com/v3/codes_of_conduct/#list-all-codes-of-conduct